src/native/eventpipe/ds-dump-protocol.c

```diff
@@ -87,7 +87,7 @@ dump_protocol_helper_unknown_command (
 	DiagnosticsIpcMessage *message,
 	DiagnosticsIpcStream *stream)
 {
-	DS_LOG_WARNING_1 ("Received unknown request type (%d)\n", ds_ipc_header_get_commandset (ds_ipc_message_get_header_ref (message)));
+	DS_LOG_WARNING_1 ("Received unknown request type (%d)", ds_ipc_header_get_commandset (ds_ipc_message_get_header_ref (message)));
```

Comment:
Are these actionable warnings for end users?  I think this could be at a lower logging level @josalem

Reply:
We are hooking things up to our trace functionality (what use for all internal runtime tracing), warnings like this will always be printed into the console by default, but you can exclude all DiagnosticServer logging if needed using Mono's trace mask.

Reply:
@lambdageek maybe we should disable MONO_TRACE_DIAGNOSTICS form default logging mask, making it an opt feature?